### DISCUSSION TITLE
NRG format file support

### DISCUSSION THREAD

**Discussion by MrPepka** [86Box/86Box#4255] [Ideas]
I have some game CD images in nrg format. Is it possible to add support for this format in 86Box? I checked the internet and although there is an article about this format in Wikipedia, I do not see documentation for it (although maybe there is one?)

**Comment by re9177** [comment]
I don't think Nero's NRG format is documented at all from what i can tell, but as an workaround if possible convert the game CD images to something like bin/cue (if it has CD audio tracks) or .iso if it's just a data disc, and then you can mount it in 86Box like normal.

**Comment by jriwanek** [comment]
There is some documentation on wikipedia, albeit not complete: https://en.wikipedia.org/wiki/NRG_(file_format)

Some code here too: https://github.com/sysprogs/WinCDEmu/tree/master/ImageFormats

**Comment by Korkman** [comment]
More importantly, WinCDEmu has code for CloneCD and Alcohol 120% Images. These image types are mostly encountered when copy protection goes beyond what can be stored with bin/cue files. I don't know how good the WinCDEmu driver is when it comes to reading those images (whether copy protection bits are properly represented), but it would be a good starting point for an implementation within 86box.

**Comment by Cacodemon345** [comment]
https://github.com/basrach/nrg2iso/blob/master/FORMAT.md

More recent-ish documentation.
